documentation/topics/security/policies.md

```diff
@@ -54,6 +54,37 @@ There are four check types, all of which do what they sound like they do:
 
 If a single check does not explicitly authorize or forbid the whole policy, then the flow moves to the next check. For example, if an `authorize_if` check does NOT return true, this _does not mean the whole policy is forbidden_ - it means that further checking is required.
 
+### Alternative Ways To Write a Policy
+
+#### Policy with keyword list
+
+```elixir
+policies do
+  policy always(), authorize_if: always()
```

Comment:
I worry a bit about adding confusion for these various syntaxes. There isn't really any good reason for someone to have to know that you can use this keyword list syntax. The `do` block syntax is preferable. I think better to leave this out here, or at the very least put this at the bottom.

Reply:
I see. I did not know there was a preference toward  `do` block. I will remove it.

In my personal project, I was using mostly the keyword list variant, because I find it easier to read. Also it reduce some lines.